dataflow/spanner-io/src/main/java/com/example/dataflow/SpannerReadApi.java

```diff
@@ -83,6 +90,32 @@ public static void main(String[] args) {
         .apply(ToString.elements())
         .apply(TextIO.write().to(options.getOutput()).withoutSharding());
 
-    p.run().waitUntilFinish();
+    pipeline.run().waitUntilFinish();
+  }
+
+  static PCollection<Struct> spannerRead(String instanceId, String databaseId, Pipeline pipeline) {
```

Comment:
Can we add method jDoc to explain the differences? 
Like the casing of the table name and the column names

Reply:
Done (here and elsewhere)